### PR TITLE
minimega: destroy namespace on remote nodes

### LINKED ISSUE #984: minimega: `clear namespace` for remote hosts
`clear namespace <namespace>` should destroy the namespace on the local node and the remote nodes.

Right now, it only destroys it on the local node which leaves VMs running on the remote node unless you use:

```
mesh send all clear namespace <namespace>
clear namespace <namespace>
```

### PR DESCRIPTION
**Pull request by jcrussell**
Fixes #984 by broadcasting out the `clear namespace <namespace>` command
to all remote nodes. Very prone to triggering #624.

### CODE REVIEW

File: doc/content/articles/developer/namespaces.article [hunk 1]
```diff
@@ -59,5 +59,5 @@ Without some mechanism to resolve this, we would fan out again and cause a
 deadlock. To prevent this, we tag the outgoing `minicli.Command` using the
 `Source` field. Specifically, we set the `Source` field to the active
 namespace. The `wrapBroadcastCLI` and `wrapVMTargetCLI` handlers check the
-`Source` field against the active namespace and, if it matches, perform the
-local behavior. Otherwise, they will fan out.
+`Source` field and, if it non-zero, perform the local behavior. Otherwise, they
```
Comment (floren) [inline doc/content/articles/developer/namespaces.article]: you accidentally a word before 'non-zero'

**Comment by jcrussell** [comment]
#624 is not fully fixed -- if you `mesh send <host> vm launch`, we'll deadlock.

**Comment by jcrussell** [comment]
You can also break things with `ns run ns run ...`.

**Comment by jcrussell** [comment]
This is similar to `mesh send all mesh send ...` -- we disallow the `mesh send` subcommand in `mesh send`. We would have to change `mesh send` to check for `ns run` as well. Perhaps it would be better to move the check to the subcommand -- some commands cannot be subcommands (e.g. `mesh send` and `ns run`).

**Comment by glattercj** [comment]
Seems to be fixed on our end.

**Pull request merged**